pm_symmetry/pm_symmetry.xml

```diff
@@ -0,0 +1,40 @@
+<tool id="pm_symmetry" name="Generate Wyckoff points symmetry report (pm-symmetry)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">pymuonsuite</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip3 show pymuonsuite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        ln -s $structure input.cell &&
+        pm-symmetry input.cell > out.txt 
+    ]]></command>
+    <inputs>
+        <param type="data" name="structure" format="cell" />
+        <param type="hidden" name="testing" value="false"/>
+    </inputs>
+    <outputs>
+        <data label="symmetry of $structure.name" name="out_sym" format="txt" from_work_dir="out.txt"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="structure" value="Si.cell" ftype="txt"/>
+            <output name="out_sym" file="test_out.txt" ftype="txt"  compare="sim_size" delta="500">
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+        usage: pm-symmetry structure
+
+        Given an input structure, generates a Wyckoff points symmetry report for that structure.
+    ]]></help>
+    <citations>
```

Comment:
Are there any paper citations needed for `pm-symmetry`? IIRC it is described in the UEP paper